app/controllers/catalog_controller.rb

```diff
@@ -240,11 +240,11 @@ def index
           contributor_text_nostem_im^3
           topic_tesim^2
 
-          exploded_project_tag_ssim^2
-          exploded_nonproject_tag_ssim
-          exploded_tag_ssim
```

Comment:
qttest fields searched:  tag_ssim is sufficient for searching.  "exploded" tag fields are for hierarchical facets